src/utils/checkAddedInCart.js

```diff
@@ -0,0 +1,13 @@
+export const checkAddedInCart = (arr, product) => {
+  if (arr.length === 0) {
+    return false;
+  }
+
+  for (let item of arr) {
+    if (item._id === product._id) {
+      return true;
+    }
+  }
+
+  return false;
```

Comment:
You can use `array.some()` method to simplify this

Reply:
Cool! Did not know about this one...updated the code using this function. Thanks!!